testing/acceptance_test.py

```diff
@@ -1297,7 +1297,11 @@ def test_simple():
     """
     )
     result = testdir.runpytest_subprocess(
-        testpath, "--capture=tee-sys", "--junitxml=output.xml"
+        testpath,
+        "--capture=tee-sys",
+        "--junitxml=output.xml",
+        "-o",
+        "junit_logging=all",
```

Comment:
Nice, the test was actually buggy: `junit_logging` by default is `no`, so it should not be writing stdout contents on the xml file. 👍